### PR TITLE
TEC-3390 Textdomain fixes

### PR DESCRIPTION
**Pull request by andrasguseo**
A number of text domains were wrong or had typos.

[TEC-3390] 

[TEC-3390]: https://moderntribe.atlassian.net/browse/TEC-3390

### CODE REVIEW

File: src/Tribe/REST/V1/Endpoints/Swagger_Documentation.php [hunk 1]
```diff
@@ -123,7 +123,7 @@ protected function get_own_documentation() {
 			'get' => array(
 				'responses' => array(
 					'200' => array(
-						'description' => __( 'Returns the documentation for The Events Calendar REST API in Swagger consumable format.', 'the-event-calendar' )
+						'description' => __( 'Returns the documentation for The Events Calendar REST API in Swagger consumable format.', 'the-events-calendar' )
```
Comment (tr1b0t) [inline src/Tribe/REST/V1/Endpoints/Swagger_Documentation.php]: * :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 5749399ec519a5a695b71476ffe14b2f</sub>

Comment (Camwyn) [inline src/Tribe/REST/V1/Endpoints/Swagger_Documentation.php]:
```suggestion
						'description' => __( 'Returns the documentation for The Events Calendar REST API in Swagger consumable format.', 'the-events-calendar' ),
```

File: src/views/v2/day/event/venue.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
 	return;
 }
 
-$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$separator            = esc_html_x( ', ', 'Address separator', 'tribe-events-calendar-pro' );
```
Comment (Camwyn) [inline src/views/v2/day/event/venue.php]:
```suggestion
$separator            = esc_html_x( ', ', 'Address separator', 'tribe-events-calendar' );
```

File: src/views/v2/list/event/venue.php [hunk 3]
```diff
@@ -20,7 +20,7 @@
 	return;
 }
 
-$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$separator            = esc_html_x( ', ', 'Address separator', 'tribe-events-calendar-pro' );
```
Comment (Camwyn) [inline src/views/v2/list/event/venue.php]:
```suggestion
$separator            = esc_html_x( ', ', 'Address separator', 'tribe-events-calendar' );
```

Comment (andrasguseo) [reply src/views/v2/list/event/venue.php]: @Camwyn then it should be 'the-events-calendar' !

**Comment by andrasguseo** [comment]
@bordoni 
> 
> Wait actually no. There are still a couple of Pro things here.

Not sure where, so I'll leave this to you.

I found 2 'events-calendar-pro' references still, but those are referring to the plugin name and not the text domain.

**Comment by lucatume** [comment]
@andrasguseo what's the status of this PR, should the assignee change to someone else?

**Comment by andrasguseo** [comment]
> 
> 
> @andrasguseo what's the status of this PR, should the assignee change to someone else?

Stephen @Camwyn told me he's going to check in with Gustavo on some of those. That is where I jumped out.

**Comment by Camwyn** [comment]
@lucatume @andrasguseo I believe most of the fixes are ready. 
I did have a brief convo with @bordoni about it and I think he and I are on the same page.
I intended to create a readme for it as well but got sidetracked - I'll do so tomorrow and get this thing moving again.

@andrasguseo - this was a **great** fix to begin with, thanks for the work! 🌮

**Pull request merged**